server/controllers/bookingRequest.js

```diff
@@ -0,0 +1,121 @@
+const asyncHandler = require("express-async-handler");
+const User = require("../models/User");
+const BookingRequest = require("../models/BookingRequest");
+
+// @route POST /bookingrequests/create
+// @desc Create a new request
+// @access Private
+
+exports.createBookingRequest = asyncHandler(async (req, res, next) => {
+  try {
+    const userId = req.user.id;
+    const user = await User.findById(userId);
+    const { sitter_id, start, end, description } = req.body;
+    if (user) {
+      const sitter = await User.findById(sitter_id);
+      if (sitter) {
+        const bookingRequest = await BookingRequest.create({
+          user_id: user,
+          sitter_id: sitter,
+          start,
+          end,
+          description,
+        });
+
+        res.status(201).json({
+          success: {
+            bookingRequest: {
+              request_id: bookingRequest._id,
+              user_id: bookingRequest.user_id._id,
+              sitter_id: bookingRequest.sitter_id._id,
```

Comment:
I don't understand why you have `user_id._id` and `sitter_id._id`

Reply:
@ezeilo-su  This is so the response only contains the id, and not the entire user or sitter object.